src/analysisd/eventinfo.c

```diff
@@ -758,6 +758,7 @@ void Zero_Eventinfo(Eventinfo *lf)
     lf->decoder_info = NULL_Decoder;
 
     lf->filename = NULL;
+    lf->mode = NULL;
```

Comment:
Please, add in the function `w_copy_event_for_log` the FIM mode field.

Reply:
Done 1a82967